### PR TITLE
Fixes #29535 - Exclude devtoolset

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: mock/el8.cfg [hunk 1]
```diff
@@ -2,7 +2,7 @@ config_opts['root'] = 'epel-8-x86_64'
 config_opts['target_arch'] = 'x86_64'
 config_opts['legal_host_arches'] = ('x86_64',)
 
-config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep --exclude=devtoolset*'
```
Comment (ekohl) [inline mock/el8.cfg:5]: I'm not sure devtoolset ships on EL8. @ehelms @pcreech @zjhuntin @evgeni any idea?

**Comment by ehelms** [comment]
Does not fix my issues :/

File: mock/el8.cfg (hunk 1 above)
Comment (pcreech) [reply mock/el8.cfg:5]: Looks like it is not available on el8 at the moment.

**Comment by ekohl** [comment]
@ehelms is there any broken cache present perhaps? Maybe remove `/var/lib/mock/*`

**Comment by ehelms** [comment]
I tried cleaning all of that. I'm good with this change going in just wanted to note it still plagues me.

File: mock/el8.cfg (hunk 1 above)
Comment (ehelms) [reply mock/el8.cfg:5]: Not to my knowledge there isn't.

**Comment by ekohl** [comment]
So with this you still get the libelf issue? https://github.com/rpm-software-management/mock/pull/268 disables sclo but not a good solution since we need that. https://github.com/rpm-software-management/mock/pull/268#issuecomment-534384263 talks about what provides it and lists `elfutils-libelf`. Perhaps we should explicitly install that to force it?

**Comment by ehelms** [comment]
Tried all of that, no dice. Maybe Fedora 32 will provide me better luck.

**Comment by stejskalleos** [comment]
@ehelms I also have problem with missing `libelf.so.1`, only fix that worked for me was adding `config_opts['yum_install_command'] += ' --disablerepo sclo*'` to `el7-scl.cfg`.

**Comment by ehelms** [comment]
@stejskalleos Oddly, I tried that tactic a week back and did not work for me. Tried it today and boom I got past the error! Thanks! Mind including that here?

**Comment by stejskalleos** [comment]
@ehelms I think we should not add it as default options, as @ekohl wrote in previous comment:

> ... disables sclo but not a good solution since we need that.

**Comment by ehelms** [comment]
@stejskalleos From all my testing, that option does not break anything and only fixes things. This has the potential to be a common issue more folks run into as newer Fedora's release and folks upgrade. I think it is safe to include it.

**Comment by ekohl** [comment]
@ehelms I thought sclo was used for the Red Hat SCLs. devtoolset comes from the RH SCLs.

**Comment by ehelms** [comment]
This doesn't appear to break my ability to build SCL packages. And I believe that is because this command is run during chroot setup and is not a permanent change within the build environment for yum itself.

**Pull request merged**